packages/std/src/math/int64.rs

```diff
@@ -494,12 +491,12 @@ mod tests {
 
     #[test]
     fn int64_new_works() {
-        let num = Int64::new([1; 8]);
+        let num = Int64::from_be_bytes([1; 8]);
         let a: [u8; 8] = num.to_be_bytes();
         assert_eq!(a, [1; 8]);
 
         let be_bytes = [0u8, 222u8, 0u8, 0u8, 0u8, 1u8, 2u8, 3u8];
-        let num = Int64::new(be_bytes);
+        let num = Int64::from_be_bytes(be_bytes);
```

Comment:
We now test `from_be_bytes` in `int64_new_works`. Could you add one test function for each of the two constructors?